iceoryx_posh/test/moduletests/test_popo_trigger.cpp

```diff
@@ -103,8 +121,8 @@ TEST_F(Trigger_test, MovedConstructedValidTriggerIsValid)
 
 TEST_F(Trigger_test, MovedAssignedValidTriggerIsValid)
 {
-    Trigger sut = createValidTrigger();
-    Trigger trigger = createValidTrigger();
+    Trigger sut = createValidStateBasedTrigger();
```

Comment:
I think you should create an invalid `sut` or invalidate it afterwards due to `Trigger() noexcept = delete`

Reply:
But I want a valid `sut` here ... I do not understand the issue here...